### ISSUE TITLE
term 'topic' not necessarily intuitive, and relation to 'scheme'

### ISSUE THREAD

**Issue by JPrevost** [MITLibraries/topichub#335]
*term 'topic' not necessarily intuitive, and relation to 'scheme'.   The group all had the same reaction, that the word 'topic' did not intuitively evoke for them the idea of searching for particular metadata search terms.  I can see though that in being very high level to work across many kinds of data and repositories, and with the name of the service being 'topic hub' that it may be terminology you need to stay with.  I'm sure we will get used to it over time.

Perhaps some small edits to the explanation screen would help-- e.g. possibly this edit:
"A topic is an identifier or term belonging to a standard vocabulary or name-space ( a piece of metadata e.g. 'mit.edu' within email domains, or 'massachusetts institute of technology within affiliation descriptions).  [then explain more how the word 'scheme' is used as that's not entirely clear to us I don't think.  ].

(from Ellen)

**Comment by richardrodgers** [comment]
While ATM I have  no inspiring alternative to 'topic', I have been toying with an alternative to 'scheme'.  The term scheme came straight out of web-addressing: the 'scheme' is the portion of a URI that describes the naming system (smtp:, http:, etc) - and by analogy ORCID is a naming scheme, email is a naming scheme, etc It gets a little stretched  in unconstrained syntax (free text) schemes like institutional names, but there scheme means that all the values come from *one place* (the MARC xxx field or whatever).

One alternative that is *slightly* less jargony (but I'd argue, still a bit library-centric) would
be 'authority'. Authority would work for ORCID (or ISSN, which we had) because there is a agency that is responsible for the values. It would also work for subject topci families like MESH, etc, Again, there is no perfect fit (that I can think of) - but authority might work well enough for email and institution name.

Whaddya think?

**Comment by richardrodgers** [comment]
'Vocabulary' also came to mind, but I think I prefer authority

**Comment by JPrevost** [comment]
Category?

**Comment by richardrodgers** [comment]
Maybe, but there can (and are) multiple schemes possible in what I think of as a category (personal identity, institution, etc). E.g. I want to add ISNI institutional IDs in addition to the names we have, and they seem like the same 'category' to me.

**Comment by JPrevost** [comment]
Good point, but I don't think Categories breaks inherently due to that... particularly as UI terminology.

I'm thinking of ORCID and how it seems like a different category of information than a person's name even if they both end up linking to People.

That said, I'm not entirely sold on Category  either :)

**Comment by richardrodgers** [comment]
How about domain or realm?

**Comment by JPrevost** [comment]
For our immediate needs, I've updated the front page blurb on the staging site using suggestions from the original report to provide examples. We can leave this open to continue to track our thoughts on Schemes. See also #351 which addresses the same problem from a different direction (i.e. show people what Schemes are with some visual examples and see if that solves the naming problem).
